docs/FAQ.md

```diff
@@ -319,22 +319,22 @@ Textual has a design system which guarantees apps will be readable on all platfo
 
 There is currently a light and dark version of the design system, but more are planned. It will also be possible for users to customize the source colors on a per-app or per-system basis. This means that in the future you will be able to modify the core colors to blend in with your chosen terminal theme.
 
-!!! Changed in 0.80.0
+!!! tip "Changed in version 0.80.0"
 
-  Textual added an `ansi_color` boolean to App. If you set this to `True`, then Textual will
-  not attempt to convert ansi colors. Note that you will lose transparency effects if you enable
-  this setting.
+    Textual added an `ansi_color` boolean to App. If you set this to `True`, then Textual will not attempt to convert ANSI colors. Note that you will lose transparency effects if you enable this setting.
 
 ---
 
-<a name="why-doesn't-the-`datatable`-scroll-programmatically"></a>
+<a name="why-doesnt-the-datatable-scroll-programmatically"></a>
```

Comment:
Did the apostrophes break something?

This file is generated automatically by [FAQTory](https://pypi.org/project/faqtory/), which means you would need to update the MD files in the questions directory.

But if nothing is broken, I'd prefer to leave the links as they are. Just in case they are in use somewhere.

Reply:
Those apostrophes were removed when I ran `faqtory build`. I am also curious as to why they were removed.

Reply:
Make sure you are running the latest version. Which I think is 1.1.3

Reply:
Fair enough, I guess we haven't rebuilt the faq in a while.